data/korath/korath ships.txt

```diff
@@ -383,89 +383,846 @@ ship "'olofez"
+		"drag" 18
+		"heat dissipation" .37
+		"solar collection" 5.6
+		"fuel capacity" 1000
+		"cargo space" 264
+		"outfit space" 856
+		"weapon capacity" 321
+		"engine capacity" 241
+		weapon
+			"blast radius" 400
+			"shield damage" 8000
+			"hull damage" 4000
+			"hit force" 12000
+	outfits
+		"Grab-Strike Turret" 1
+		"Banisher Grav-Turret" 3
+		"Warder Anti-Missile" 2
+		"Digger Mining Beam" 2
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Fuel Processor"
+		"Brig"
+		"Microbot Defense Station" 17
+		"Thermal Repeater Rifle" 272
+		
+		"Thruster (Stellar Class)"
+		"Steering (Planetary Class)"
+		"Scram Drive"
+	
+	engine -37 333 .8
+	engine 0 333 .8
+	engine 37 333 .8
+	"reverse engine" -19 -303 0.5
+		over
+	"reverse engine" 0 -303 0.5
+		over
+	"reverse engine" 19 -303 0.5
+		over
+	fighter -90 160 back
+	fighter 0 160 back
+	fighter 90 160 back
+	gun -45 -253 "Digger Mining Beam"
+	gun 45 -253 "Digger Mining Beam"
+	turret -77 -116.5 "Grab-Strike Turret"
+	turret 77 -116.5 "Warder Anti-Missile"
+	turret -87 -83 "Banisher Grav-Turret"
+	turret 87 -83 "Banisher Grav-Turret"
+	turret -51 276.5 "Warder Anti-Missile"
+	turret 51 276.5 "Banisher Grav-Turret"
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "The largest of Korath production vessels are derived from a common and ancient chassis. This bulbous ship has been adapted to carry passengers between stations, or boarding parties into enemy strongholds."
+
+
+ship "Ra'at Ik 621"
+	sprite "ship/world-ship e"
+	thumbnail "thumbnail/world-ship e"
+	attributes
+		category "Heavy Warship"
+		cost 33590000
+		shields 48000
+		hull 31400
+		"required crew" 794
+		"bunks" 1037
+		"mass" 1762
+		"drag" 27
+		"heat dissipation" .41
+		"fuel capacity" 1000
+		"cargo space" 228
+		"outfit space" 999
+		"weapon capacity" 407
+		"engine capacity" 277
+		weapon
+			"blast radius" 400
+			"shield damage" 8000
+			"hull damage" 4000
+			"hit force" 12000
+	outfits
+		"Grab-Strike Turret" 3
+		"Banisher Grav-Turret" 2
+		"Warder Anti-Missile" 2
+		"Fire-Lance" 3
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Fuel Processor"
+		"Microbot Defense Station" 16
+		"Thermal Repeater Rifle" 308
+		
+		"Reverser (Lunar Class)"
+		"Thruster (Stellar Class)"
+		"Steering (Stellar Class)"
+		"Jump Drive"
+	
+	engine -42 343 .6
+	engine -22 343 .6
+	engine -22 343 .6
+	engine 22 343 .6
+	engine 22 343 .6
+	engine 42 343 .6
+	"reverse engine" -16 -336 0.9
+	"reverse engine" 16 -336 0.9
+	fighter -100 -169
+	fighter 0 -169
+	fighter 100 -169
+	fighter -98 156 back
+	fighter 98 156 back
+	fighter -50 303 back
+	fighter -50 303 back
+	fighter 50 303 back
+	fighter 50 303 back
+	drone -100 -169
+	drone 100 -169
+	drone -98 156 back
+	drone 98 156 back
+	drone -50 303 back
```

Comment:
```suggestion
	bay "Fighter" -100 -169
	bay "Fighter" 0 -169
	bay "Fighter" 100 -169
	bay "Fighter" -98 156 back
	bay "Fighter" 98 156 back
	bay "Fighter" -50 303 back
	bay "Fighter" -50 303 back
	bay "Fighter" 50 303 back
	bay "Fighter" 50 303 back
	bay "Drone" -100 -169
	bay "Drone" 100 -169
	bay "Drone" -98 156 back
	bay "Drone" 98 156 back
	bay "Drone" -50 303 back
```
Swap for new keyword for bays, `back` probably should be on line down with another indent as well but haven't find way to type tabs on phone yet.